_sass/contributor.scss

```diff
@@ -37,7 +37,7 @@
 			}
 		}
 	.name {
-		font-size: calc(.8em + 1.4vw);
+		font-size: calc(.75em + 1.1vw);
```

Comment:
Nice work @PluckyPrecious But instead of changing the font size of the names you could reduce the size of the `GitHub` and `Twitter` since the name is more important than those two. 

```
.twitter,
	.github {
		font-size: calc(0.35em + 1.1vw);
		margin-top: 0.5vw;
		white-space: nowrap;
		img {
			border-radius: 4px; /* Rounded border */
			width: 20px; /* Set a small width */
			vertical-align: middle;
		}
	}
```